src/react/account/AccountUserPoliciesList.tsx

```diff
@@ -0,0 +1,306 @@
+import React, { ChangeEvent, useMemo, useState } from 'react';
+import { useHistory, useRouteMatch } from 'react-router-dom';
+import { Table, Button } from '@scality/core-ui/dist/next';
+import { spacing } from '@scality/core-ui/dist/style/theme';
+import { formatShortDate } from '../utils';
+import { useIAMClient } from '../IAMProvider';
+import { useAwsPaginatedEntities } from '../utils/IAMhooks';
+import { TitleRow as TableHeader } from '../ui-elements/TableKeyValue';
+import CopyARNButton from '../ui-elements/CopyARNButton';
+import { useQueryParams } from '../utils/hooks';
+import SearchInputComponent from '@scality/core-ui/dist/components/searchinput/SearchInput.component';
+import { Tooltip } from '@scality/core-ui';
+import SpacedBox from '@scality/core-ui/dist/components/spacedbox/SpacedBox';
+import { notFalsyTypeGuard } from '../../types/typeGuards';
+import { useMutation, useQuery } from 'react-query';
+import { queryClient } from '../App';
+import DeleteConfirmation from '../ui-elements/DeleteConfirmation';
+import { Banner } from '@scality/core-ui';
+import { getUserPoliciesQuery } from '../queries';
+import { Icon } from '../ui-elements/Help';
+
+const RenderEditButton = ({ policyName }: { policyName: string }) => {
+  const history = useHistory();
+  const disableCondition = ['storage-account-owner-policy', 'storage-manager-policy'].includes(policyName);
```

Comment:
I would suggest defining a constant for 'storage-account-owner-policy', 'storage-manager-policy'